common/app/views/fragments/image.scala.html

```diff
@@ -10,6 +10,19 @@
     isImmersiveMainMedia: Boolean = false
 )
 
+@immersivePortraitSource(largestImage: model.ImageAsset, hidpi: Boolean = false) = {
+    <source media="@if(hidpi) {
+        (orientation: portrait) and (-webkit-min-device-pixel-ratio: 1.25), (orientation: portrait) and (min-resolution: 120dpi)
+    } else {
+        (orientation: portrait)
+    }"
+    sizes="@{100 * (largestImage.width.toDouble / largestImage.height.toDouble)}vh"
+    srcset="@{(500 to 4000 by 250)
```

Comment:
> Does it mean we will be providing sources even over the specific image’s maximal resolution

That is a good point! I am making a change now that will increment by 250 until the largest image width, rather than by 4000